### DISCUSSION TITLE
Stage 3: Weather Data Api & Weather Cache Implementation

### DISCUSSION THREAD

**Discussion by sanket1211996** [airavata-courses/SuperNova#59] [Ideas]
I have opened up the stage 3 for weather data implementation discussion.

**Comment by richajha012** [comment]
I have created PR #67 for issue weather API implementation. #5 

Please do review and merge

**Comment by sumedhsalvi** [comment]
I have created PR #76 for issue #75 - cache prototype implementation using Redis key-value store and tested using fake API.
Please review and merge.

**Reply by sanket1211996** [reply]
Good Job, PR Merged.

**Comment by sumedhsalvi** [comment]
I have created PR #78  for issue #77 - weather plot endpoint implementation in cache microservice.
Please review and merge.

**Reply by sanket1211996** [reply]
Approved and Merged.

**Comment by sumedhsalvi** [comment]
I have created PR #80 for issue #79 - query status endpoint implementation in cache microservice.
Please review and merge.

**Reply by sanket1211996** [reply]
Approved and Merged.

**Comment by sanket1211996** [comment]
I have added cache integration changes #74. Do review and merge.

**Comment by sumedhsalvi** [comment]
I have created PR #87 for issue #86 - Weather data status in cache and added body elements in query status endpoint.
Please review and merge.

**Reply by sanket1211996** [reply]
Done

**Comment by sanket1211996** [comment]
I have also Dockerized Weather Cache #88 , Dockerized User APi Cache #85 , Dockerized Weather Api #84, Dockerized Gateway . Review and merged.
